CHANGELOG.md

```diff
@@ -2,6 +2,49 @@
 
 ## Next release
 
+## Mypy 1.13
+
+We’ve just uploaded mypy 1.13 to the Python Package Index ([PyPI](https://pypi.org/project/mypy/)).
+Mypy is a static type checker for Python. You can install it as follows:
+
+    python3 -m pip install -U mypy
+
+You can read the full documentation for this release on [Read the Docs](http://mypy.readthedocs.io).
+
+Note that unlike typical releases, Mypy 1.13 does not have any changes to type checking semantics
+from 1.12.1.
+
+### Improved performance
+
+Mypy 1.13 contains several performance improvements. Users can expect mypy to be 5-20% faster.
+In environments with long search paths (such as environments using many editable installs), mypy
+can be significantly faster, e.g. 2.2x faster in the use case targeted by these improvements.
+
+Mypy 1.13 allows use of the `orjson` library for handling the cache instead of the stdlib `json`,
+for improved performance. You can ensure the presence of `orjson` using the `faster-cache` extra:
```

Comment:
are there any stats for this? Any downsides? What's the comparison with sqlite?

Reply:
Depends on your workload and whether incremental or not, Jukka measured 10-15% in https://github.com/python/mypy/pull/17955#issuecomment-2413377053 , I've seen the lower end of that

It composes with sqlite, since mypy's sqlite thing just dumps json into sqlite. I think sqlite cache is mainly a way to workaround filesystem performance. I haven't benchmarked (or really used) the sqlite option. Jukka mentions he'd like to make it the default in the future though: https://github.com/python/mypy/pull/17955#issuecomment-2413270549